src/components/SingleMovie.js

```diff
@@ -1,15 +1,31 @@
 import React from "react"
+import "../css/SingleMovie.css"
 
 function SingleMovie ({movieInFocus, movies}) {
 
   const correctMovie = movies.find(movie => movie.id === movieInFocus)
 
   return (
-    <div>
-      <img src={correctMovie.poster_path} height="300" width="200"></img>
-      <p>Title: {correctMovie.title}</p>
-      <p>Rating: {correctMovie.average_rating}</p>
-      <p>Release Date {correctMovie.release_date}</p>
+    <div className="SingleMovie" style={{
```

Comment:
In a future refactor, this might be a cool thing to just add to the SingleMovie stylesheet instead of inline styling. It's kind of a big style object.

Reply:
I'll just +1 to this point.  In general, try to keep most styles inside of a CSS file.  That being said, since you're trying to make the `backgroundImage` dynamic based on the incoming data, I think it's to fine to keep that specific style here.